version.cmake

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) 2017-2018 Prism Skylabs
-set(ConnectSDK_VERSION_MAJOR 1)
-set(ConnectSDK_VERSION_MINOR 14)
-set(ConnectSDK_VERSION_REVISION 1)
+set(ConnectSDK_VERSION_MAJOR 2)
```

Comment:
After this change, VC won't work with the Retail system. Shouldn't we fork the repo from this point?

Reply:
Yes, we are forking to 'prismai_connect'